index.html

```diff
@@ -539,6 +538,17 @@ <h3>
           indicated by the <a>active</a> attribute changes.
         </p>
       </section>
+      <section data-dfn-for="WakeLockRequestOptions"> <h3>The <dfn>WakeLockRequestOptions</dfn> dictionary</h3>
+        <pre class="idl">
+          dictionary WakeLockRequestOptions {
+            AbortSignal? signal;
+          };
+        </pre>
+        <p>
+          The <dfn>signal</dfn> property is an {{AbortSignal}} used to abort
```

Comment:
Avoid restating things that are in the IDL (i.e., the type).

```suggestion
          The <dfn>signal</dfn> member is used to abort
```